third_party/zip_files.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
Actually, ijar already implements zipping/unzipping. We could use that -- but that requires plumbing that's out of scope for this PR.

For now I request no changes here.

Reply:
Thanks Laszlo for the review! I chose to implement the scripts in bash because the setup takes less time and is shorter than in Java. I am not at all familiar with Python so I probably have to invest some time before writing a script like this. These scripts are also only used by a Buildkite pipeline to release the Java tools, in which case toolchain definition is not that relevant.